.pipelines/containers/container-template-windows.yaml

```
@@ -13,9 +13,10 @@ steps:
 
 - powershell: |
     echo "using tag ${{ parameters.tag }}"
-    powershell.exe -command "& { . .\build\scripts\windows.ps1; Retry({${{ parameters.name }}-image windows-${{ parameters.arch }}-${{ parameters.tag }}}) }"
+    powershell.exe -command "& { . .\build\scripts\windows.ps1; ${{ parameters.name }}-image windows-${{ parameters.arch }}-${{ parameters.tag }} }"
```

don't think we need the subshell invocation here:
```suggestion
    . .\build\scripts\windows.ps1; ( ${{ parameters.name }}-image windows-${{ parameters.arch }}-${{ parameters.tag }} )
```